config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -137,23 +105,23 @@ periodics:
           memory: "8Gi"
 
 # manual-release-bump-required
-- interval: 30m
+- interval: 1h
   name: ci-kubernetes-build-stable1
   labels:
     preset-service-account: "true"
     preset-dind-enabled: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/bootstrap:v20190211-fab6034b4
+    - image: gcr.io/k8s-testimages/bootstrap:v20190213-2f4f06285
       args:
-      - --repo=k8s.io/kubernetes=release-1.12
+      - --repo=k8s.io/kubernetes=release-1.13
       - --repo=k8s.io/release
       - --root=/go/src
       - --timeout=180
       - --scenario=kubernetes_build
       - --
       - --allow-dup
-      - --extra-publish-file=k8s-stable1
+      - --extra-publish-file=k8s-beta
```

Comment:
job name references `stable1` but this is referencing `beta`, I'm also unclear on why the interval bump

Reply:
good catch... here are some bad changes going on :joy:

Reply:
I think @amwat changed the job name and added a new one on the top? fixing..

Reply:
done